src/PrestaShopBundle/Translation/Extractor/ThemeExtractor.php

```diff
@@ -93,6 +95,35 @@ public function extract(Theme $theme, $locale = 'en-US', $rootDir = false)
         throw new \LogicException(sprintf('The format %s is not supported.', $this->format));
     }
 
+    private function overrideFromDefaultCatalog($locale, &$catalogue)
+    {
+        $defaultCatalogue = $this->themeProvider
+            ->setLocale($locale)
+            ->getDefaultCatalogue()
+        ;
+
+        if (!empty($defaultCatalogue)) {
+            $defaultCatalogue = current($defaultCatalogue);
```

Comment:
Yes, maybe it's the first, i don't want to change the behavior. I just do like other time in translation context :/